### PR TITLE
docs: ui capabilities text field PTL-1581

### PR DESCRIPTION
**Pull request by Gareth-Spencer-Genesis**
ATENTION! YOU SHOULD NOW BRANCH FROM PREPROD WHEN YOU UPDATE 

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No

Have you done a trial build to check all new or changed links?
Yes

Is there anything else you would like us to know?
No

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: examples/ui/client-capabilities/interaction/text-field.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { CodeLabel, CodeSection } from '../../documentationBase';
+import { registerComponents } from '../../rapidImports';
+
+registerComponents();
```
Comment (patrickoneill-genesis) [inline examples/ui/client-capabilities/interaction/text-field.js:4]: add the fix for this import

File: docs/001_develop/03_client-capabilities/008_interaction/text-field.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+    Declaration
+    ```html
+    <rapid-text-field></rapid-text-field>
+    ```
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleTextFieldChanged = (event) => {
+        console.log({ event.target.value })
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/008_interaction/text-field.mdx:71]: ` console.log(event.target.value)`

File: examples/ui/client-capabilities/interaction/text-field.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+export default function TextFieldDemo({ children, color }) {
+	return (<CodeSection><rapid-text-field placeholder="Input text here">Sample Text Field</rapid-text-field></CodeSection >)
```
Comment (patrickoneill-genesis) [inline examples/ui/client-capabilities/interaction/text-field.js:7]: can we have one each of 
type
{"email", "password","tel","number","text","url"}   

One 'start' slot with a `$` sign
One 'end' slot with a '%' sign

File: examples/ui/client-capabilities/interaction/text-field.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	return (
+		<CodeSection>
+
+
+			<div style={{ color: 'var(--neutral-foreground-rest)', width: '-webkit-fill-available'}}>
+				<div style={{ display: 'flex', flexWrap: 'wrap', gap: '10px', flexDirection: 'column', }}>
+
+					<rapid-text-field placeholder="Sample text" type={"text"}>Text Field</rapid-text-field>
+
+					<rapid-text-field placeholder="Format: .+@example\.com" type={"email"} pattern=".+@example\.com">Email Text Field</rapid-text-field>
```
Comment (patrickoneill-genesis) [inline examples/ui/client-capabilities/interaction/text-field.js:22]: the `type` attributes don't need any special syntax as they are just static strings - can just be `type="email|password|tel" etc...

**Review by wjhendry — APPROVED** [review]

**Comment by wjhendry** [comment]
@patrickoneill-genesis I'm ready to merge this and we can do further edits once it is in place

**Pull request merged**